### PR TITLE
Add contribution documentation

### LINKED ISSUE #11: Create a CONTRIBUTING.md
We can use our boilerplate; it should explain branching policies, code of conduct, etc etc etc.

### PR DESCRIPTION
**Pull request by slifty**
Now, finally, we will know how to contribute for once.

Closes #11

### CODE REVIEW

**Comment by slifty** [comment]
@reefdog I'm not going to make you review this (according to our CONTRIBUTION.md this didn't even need to be a PR)

File: docs/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+## Scope
+
+We expect people to maintain these standards both within project spaces and at events where they are representing the project or its community. Examples of representing a project or community include using an official project e-mail address, posting via project social media accounts, or acting as an appointed representative at an event.
+
+## Meeting Our Responsibilities
+
+Contributors are welcome and encouraged to resolve conflicts directly with each other, when they feel comfortable doing so.
+
+If issues can not be resolved directly among affected people, they should report instances of abusive, harassing, or otherwise unacceptable behavior to the BIFFUD Beastmaster team at **`beastmaster@biffud.com`**.
```
Comment (reefdog) [inline docs/CODE_OF_CONDUCT.md]: had this been reviewed a reviewer may have noted this graf

Comment (slifty) [reply docs/CODE_OF_CONDUCT.md]: I decided to keep that intentionally!  It's a BIFFUD value add!

Comment (reefdog) [reply docs/CODE_OF_CONDUCT.md]: haha I just got to https://github.com/TechAndCheck/tech-and-check-alerts/pull/16/files#diff-6a3371457528722a734f3c51d9238c13R126 and realized this was likely the case

**Pull request merged**